setupenv.sh

```diff
@@ -3,4 +3,4 @@ unset PYTHONPATH
 virtualenv -p python3 env
 echo "unset PYTHONPATH" >> env/bin/activate
 . env/bin/activate
-pip install requests tornado git+https://github.com/WIPACrepo/rest-tools
+pip install requests tornado pytest pytest-mock pytest-cov git+https://github.com/WIPACrepo/rest-tools
```

Comment:
Sorry, I haven't been updating `setupenv.sh` very well.
I'll make it a point to update this file if I add any dependencies to the project.